drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -524,8 +521,13 @@ function DragAndDropEditBlock(runtime, element, params) {
                             if (response.result === 'success') {
                                 window.location.reload(false);
                             } else {
-                                $('.xblock-editor-error-message', element)
-                                    .html(gettext('Error: ') + response.message);
+                                var errorMsg = $("<p></p>").text(gettext('Error: '));
+                                $('.xblock-editor-error-message', element).append(errorMsg);
```

Comment:
Note that previous error messages are never cleared out (save with error, save again with same or different error).